packages/boot/test/bootstrapTests/test-liquidation-concurrent-1.ts

```diff
@@ -0,0 +1,557 @@
+    },
+    {
+      give: '90IST',
+      price: 10.0,
+    },
+    {
+      give: '166.675IST',
+      discount: 0.15,
+    },
+  ],
+  price: {
+    starting: 13.34,
+    trigger: 10.99,
+  },
+  auction: {
+    start: {
+      collateral: 45,
+      debt: 336.675,
+    },
+    end: {
+      collateral: 9.970236,
+      debt: 0,
+    },
+  },
+};
+
+const setups = {
+  ATOM: atomSetup,
+  STARS: starsSetup,
+};
+
+const atomOutcome = /** @type {const} */ {
+  bids: [
+    {
+      payouts: {
+        Bid: 0,
+        Collateral: 8.897786,
+      },
+    },
+    {
+      payouts: {
+        Bid: 0,
+        Collateral: 10.01001,
+      },
+    },
+    {
+      payouts: {
+        Bid: 10.46,
+        Collateral: 16.432903,
+      },
+    },
+  ],
+  reserve: {
+    allocations: {
+      ATOM: 0.309852,
+    },
+    shortfall: 0,
+  },
+  vaultsSpec: [
+    {
+      locked: 3.373,
+    },
+    {
+      locked: 3.024,
+    },
+    {
+      locked: 2.792,
+    },
+  ],
+  vaultsActual: [
+    {
+      locked: 3.525747,
+    },
+    {
+      locked: 3.181519,
+    },
+    {
+      locked: 2.642185,
+    },
+  ],
+};
+
+const starsOutcome = /** @type {const} */ {
+  bids: [
+    {
+      payouts: {
+        Bid: 0,
+        Collateral: 8.08,
+      },
+    },
+    {
+      payouts: {
+        Bid: 0,
+        Collateral: 9.099181,
+      },
+    },
+    {
+      payouts: {
+        Bid: 0,
+        Collateral: 17.48,
+      },
+    },
+  ],
+  reserve: {
+    allocations: {
+      STARS: 0.306349,
+    },
+    shortfall: 0,
+  },
+  vaultsSpec: [
+    {
+      locked: 3.373,
+    },
+    {
+      locked: 3.024,
+    },
+    {
+      locked: 2.792,
+    },
+  ],
+  vaultsActual: [
+    {
+      locked: 3.525747,
+    },
+    {
+      locked: 3.181519,
+    },
+    {
+      locked: 2.642185,
+    },
+  ],
+};
+
+const outcomes = {
+  ATOM: atomOutcome,
+  STARS: starsOutcome,
+};
+
+test.before(async t => {
+  t.context = await makeLiquidationTestContext(t);
+  await addSTARsCollateral(t);
+});
+test.after.always(t => {
+  return t.context.shutdown && t.context.shutdown();
+});
+
+// Reference: Flow 1 from https://github.com/Agoric/agoric-sdk/issues/7123
+const checkFlow1 = async (t: ExecutionContext<LiquidationTestContext>) => {
+  // fail if there are any unhandled rejections
+  process.on('unhandledRejection', (error: Error) => {
+    t.fail(error.message);
+  });
+
+  const {
+    advanceTimeBy,
+    advanceTimeTo,
+    agoricNamesRemotes,
+    check,
+    setupStartingState,
+    priceFeedDrivers,
+    readLatest,
+    walletFactoryDriver,
+  } = t.context;
+
+  const cases = [
+    { collateralBrandKey: 'ATOM', managerIndex: 0 },
+    { collateralBrandKey: 'STARS', managerIndex: 1 },
+  ];
+
+  const metricsPaths = cases.map(
+    ({ managerIndex }) =>
+      `published.vaultFactory.managers.manager${managerIndex}.metrics`,
+  );
+
+  const buyer = await walletFactoryDriver.provideSmartWallet('agoric1buyer');
+
+  const setupVault = async (
```

Comment:
this sets up many vaults
```suggestion
  const setupVaults = async (
```